### ISSUE TITLE
WCAG issues on Modal component

### ISSUE THREAD

**Issue by acbteam** [ant-design-blazor/ant-design-blazor#3292]
https://github.com/ant-design-blazor/ant-design-blazor/blob/61bfb037a3d394b36d159ee068d9d208ff83903a/components/modal/core/Dialog.razor#L20

Hi!  I need to pass [WCAG 'A' validation](https://www.w3.org/TR/WCAG21/) to my site and I have found 2 issues using this modal component dialog:

1. aria-hidden="true" is used on a focusable element. (WCAG 2.1 Success Criterion: A 2.1.1 Keyboard )  

`<div id="Ant-Design-9efb3e8a-38b6-41dc-b7d6-f5e9d786b26c" tabindex="0" aria-hidden="true" style="width: 0px; height: 0px; overflow: hidden; outline: none;"></div>
`

=> We can't add an aria-hidden="true" to an active element

2. The modal dialog does not have an accessible name. (WCAG 2.1 Success Criterion:A 4.1.2 Name, Role, Value ) 

`<div tabindex="-1" class="ant-modal-wrap" role="dialog" style=""></div>`

=> Elements with role="dialog' need an arial-label attribute

**Comment by ElderJames** [comment]
Thanks for contacting us @acbteam , we need to increase the Accessibility of each component
Fit? Are you interested in contribution?
